utils/src/main/kotlin/Utils.kt

```diff
@@ -547,7 +547,7 @@ fun String.fileSystemEncode() =
  */
 fun String.isValidUrl() =
         try {
-            URL(this).toURI()
+            URL(this)
```

Comment:
You could add to the commit message that `toURI` also checks for RFC 2396 compatibility which is not intended here.

Reply:
You're right, but I'd prefer to not retrigger CI and get this merged as I have some changes based on this PR.